internal/charmstore/migrations.go

```diff
@@ -138,6 +142,36 @@ func addSupportedSeries(db StoreDatabase) error {
 	return nil
 }
 
+// handleDevelopmentEntities adds the Development field to all entities and
+// sets up ACLs for development revisions.
+func handleDevelopmentEntities(db StoreDatabase) error {
+	logger.Infof("adding development field to all entities")
+	if _, err := db.Entities().UpdateAll(nil, bson.D{{
+		"$set", bson.D{{"development", false}},
+	}}); err != nil {
+		return errgo.Notef(err, "cannot add development field to all entities")
+	}
+
+	logger.Infof("adding development ACLs to all base entities")
+	baseEntities := db.BaseEntities()
+	var baseEntity mongodoc.BaseEntity
+	iter := baseEntities.Find(bson.D{{
+		"developmentacls", bson.D{{"$exists", false}},
```

Comment:
It should be possible to do this without retrieving every document locally, but it isn't so... moan